File: app/src/main/java/com/peteralexbizjak/chatappandroid/utils/RandomColorGenerator.java
package com.peteralexbizjak.chatappandroid.utils;

import android.graphics.Color;

import java.util.Random;

public final class RandomColorGenerator {
    public static int generateRandomColorInt() {
        Random random = new Random();
        return Color.argb(200, random.nextInt(256), random.nextInt(256), random.nextInt(256));
    }
}

File: app/src/main/java/com/peteralexbizjak/chatappandroid/utils/RandomColorGenerator.kt
package com.peteralexbizjak.chatappandroid.utils

import android.graphics.Color

import java.util.Random

object RandomColorGenerator {
    fun generateRandomColorInt(): Int {
        val random = Random()
        return Color.argb(200, random.nextInt(256), random.nextInt(256), random.nextInt(256))
    }
}
